mcstasscript/tests/test_McStasPlotOptions.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+
+from mcstasscript.data.data import McStasPlotOptions
+
+
+class TestMcStasPlotOptions(unittest.TestCase):
+    """
+    Various test of McStasPlotOptions class
+    """
+
+    def test_McStasPlotOptions_default_log(self):
+        """
+        Test that newly created McStasMetaData has correct type
+        """
+        plot_options = McStasPlotOptions()
+        self.assertIsInstance(plot_options.log, bool)
+        self.assertFalse(plot_options.log)
+
+    def test_McStasPlotOptions_default_orders_of_mag(self):
+        """
+        Test that newly created McStasMetaData has correct type
```

Comment:
These docstrings all the same for all tests. Should be updated.

Reply:
I have expanded the testing of McStasPlotOptions (only some options were tested), and updated their doc strings.
In addition a new file was added, test_Plotter.py, which adds 600 lines of testing for the plotting functionality, including running the internal plotter function (without plt.show() ) for all 1D/2D and log mode combinations.

This could obviously still be improved in the sense that actual plotting is not tested or visually compared to a known correct plot.